src/Http/Http.Abstractions/src/Metadata/IDisableHttpMetricsMetadata.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Http.Metadata;
+
+/// <summary>
+/// A marker interface which can be used to identify metadata that disables HTTP request duration metrics.
+/// </summary>
+public interface IDisableHttpMetricsMetadata
```

Comment:
What's the flag interface for?  Relatedly, if there's only one concrete implementation, can we just use that?

Reply:
Using interfaces for endpoint metadata is the normal pattern. The only implementation is an attribute. Internally, builder methods that add this metadata to the endpoint just add the attribute.